core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java

```diff
@@ -148,6 +154,16 @@ public String getHealthMonitorId(){
     }
 
     @Override
+    public String getSubnetId() {
```

Comment:
So if you look at this documentation a lot of it was just copy-pasted from the old stuff. check out http://developer.openstack.org/api-ref/networking/v2-ext/index.html?expanded=show-pool-details-detail,id370-detail ( /v2.0/lb/pools/{pool_id}) its exactly the same text. And the pool object has changed. You can see that the new object in this documentation doesn't even contain the 'listener' field. I would refrain from using this documentation. Instead, use https://wiki.openstack.org/wiki/Neutron/LBaaS/API_2.0 (although even this isn't perfect)